src/Draco.Compiler/Internal/Symbols/ITypedSymbol.cs

```diff
@@ -9,4 +9,9 @@ internal interface ITypedSymbol
     /// The type of value the symbol references.
     /// </summary>
     public TypeSymbol Type { get; }
+
+    /// <summary>
+    /// Specifying if given symbol is static.
+    /// </summary>
+    public bool IsStatic { get; }
```

Comment:
Maybe this should be split out into an `IMemberSymbol`? It leaks into symbols where it's irrelevant and it's not really nice